helpers/crud/movimientos_contables/movimiento.go

```diff
@@ -55,3 +55,29 @@ func PostMovimiento(in interface{}, out interface{}) (outputError map[string]int
 	}
 	return
 }
+
+func GetMovimientosWorker(id string, conMovimientos bool, c chan interface{}) {
+	if conMovimientos {
+		query := fmt.Sprintf("TransaccionId:%v", id)
+		fields := []string{
+			"Activo",
+			"CuentaId",
+			"Descripcion",
+			"Id",
+			"NombreCuenta",
+			"TerceroId",
+			"TipoMovimientoId",
+			"Valor",
+		}
+		var movimientos interface{}
+		outputError := GetMovimientos(query, fields, -1, 0, &movimientos)
+		if outputError != nil {
+			c <- nil
```

Comment:
De manera similar a lo comentado en la L42 de `helpers/crud/cuentas_contables/comprobante.go`, ajustar:
```suggestion
			logs.Warn(outputError)
			c <- nil
```
(No aplicar este _suggested change_ directamente, ajustar según sea pertinente)